src/Equinor.ProCoSys.IPO.WebApi/appsettings.json

```diff
@@ -9,11 +9,6 @@
   "ConnectionStrings": {
     "IPOContext": ""
   },
-  "Meetings": {
```

Comment:
@llykkelarsen, should these changes just be local?

Reply:
no pretty sure they should be checked in